docs/SIGNALS.md

```diff
@@ -21,18 +21,17 @@ processes are documented here as well.
 
 ### Worker Processes
 
-Note: the master uses a pipe to signal workers
+Note: the monitor uses a pipe to signal workers
```

Comment:
Technically this is also outdated, as it is now a socketpair, but functionally that doesn't change much.

Reply:
In this PR I’d like to focus just on simple renaming, and keep other doc updates for another PR. 😅